src/skel/glfw/glfw.cpp

```diff
@@ -484,6 +561,9 @@ psInitialize(void)
 	_dwMemAvailPhys = (uint64_t)(vm_stat.free_count * page_size);
 	debug("Physical memory size %llu\n", _dwMemAvailPhys);
 	debug("Available physical memory %llu\n", size);
+#elif defined (__SWITCH__)
+	svcGetInfo(&_dwMemAvailPhys,     7, 0xffff8001, 0);
```

Comment:
https://switchbrew.github.io/libnx/svc_8h.html
```suggestion
	svcGetInfo(&_dwMemAvailPhys,     7, CUR_PROCESS_HANDLE, 0);
```

Reply:
Yeah, these are the intended defines, thanks. I actually grabbed the values from the switchbrew wiki but didn't knew they were available as defines already.